src/test/java/com/wikia/webdriver/common/core/CommonExpectedConditions.java

```diff
@@ -43,7 +43,7 @@ public Boolean apply(WebDriver from) {
         try {
           String elementsAttributeValue = findElement(locator, from).getAttribute(attribute);
           return elementsAttributeValue.contains(value);
-        } catch (StaleElementReferenceException e) {
+        } catch (StaleElementReferenceException | NullPointerException e) {
```

Comment:
getAttribute(attribute) returns null if no attribute is found - see get attribute implementation. In that case elementsAttributeValue.contains(value) gives null pointer